bigtop-deploy/puppet/modules/spark/manifests/init.pp

```diff
@@ -255,8 +255,9 @@
       require => Package['spark-core'],
     }
 
-    file { '/etc/spark/conf/log4j.properties':
-      source  => '/etc/spark/conf/log4j.properties.template',
+    # BIGTOP-4077 modify from log4j.properties.template to log4j2.properties.template
```

Comment:
@MacChen01 You can remove the comment because the intent of the manifest is obvious.

Reply:
@iwasakims 
ok, the comment has been removed, thanks.